semcore/d3-chart/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.58.5] - 2024-12-20
+
+### Added
+
+- `aria-hidden="true"` to `PatternSymbol` inside `LegendItem` to improve a11y.
```

Comment:
I think we should listen to the linter and write "A11y" capitalized :)

Reply:
capitalized :)